dateutil/test/test_isoparser.py

```diff
@@ -399,7 +407,7 @@ def test_parse_isodate(d, dt_fmt, as_bytes):
     ('2014-04-19T', ValueError),                # Unknown components
 ])
 def test_isodate_raises(isostr, exception):
-    with pytest.raises(exception):
+    with pytest.raises(exception):              #pragma: no cover
```

Comment:
This should be covered, it is a normal test that is testing that these things raise errors.